pkg/nodejs/npm/parse.go

```diff
@@ -253,6 +260,11 @@ func pkgNameFromPath(path string) string {
 	// node_modules/string-width
 	// node_modules/string-width/node_modules/strip-ansi
 	paths := strings.Split(path, "/")
+	// deps starting from `@` have pgkName with one `/`
+	// e.g. path == `node_modules/@babel/plugin-transform-classes` => pkgName == `@babel/plugin-transform-classes`
+	if strings.HasPrefix(paths[len(paths)-2], "@") {
```

Comment:
It should not happen, but I think we should check the length just in case.

```suggestion
	if len(paths) >= 2 && strings.HasPrefix(paths[len(paths)-2], "@") {
```

Reply:
Done